apps/course-authoring/src/main/preload.ts

```diff
@@ -27,3 +27,10 @@ contextBridge.exposeInMainWorld('electron', {
     },
   },
 });
+
+const exportScormApi = {
```

Comment:
We already have the `electron` object being exposed to the main world @3 so I think we should get rid of this `exportScormApi` and the `contextBridge` below and add a more generic `invoke` method to the `ipcRenderer` object in the `contextBridge.exposeInMainWorld('electron', something like:
```js
    invoke(channel: string, ...args: unknown[]) {
      const validChannels = ['ipc-example', 'start-scorm-export-process'];
      if (validChannels.includes(channel)) {
        return ipcRenderer.invoke(channel, ...args);
      }
    },
```
This way we can use it in the renderer process like this:
```js
          onClick={async () => {
            const messageFromMain = await window.electron.ipcRenderer.invoke(
              'start-scorm-export-process',
              { courseName: 'Test Course' }
            );

            console.log(messageFromMain);
          }}
```
And get the same results without using the `contextBridge` twice and attaching two different new objects to the window.